decimal64.go

```diff
@@ -457,7 +457,13 @@ func numDecimalDigits(n uint64) int {
 
 // propagateNan returns true if flavor1 should be propagated else false if flavor2 should be propagated
```

Comment:
is this comment out of date?

seems like the func always returns a `*Decimal64` (either `dp.dec`, or `ep.dec`)